gotthard/cli.py

```diff
@@ -307,7 +305,11 @@ def get_spilo_stacks(cf, spilo_cluster):
                     'UPDATE_ROLLBACK_IN_PROGRESS', 'UPDATE_ROLLBACK_FAILED',
                     'UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS', 'UPDATE_ROLLBACK_COMPLETE']
 
-    for stack in cf.list_stacks(StackStatusFilter=stack_status)['StackSummaries']:
+    paginator = cf.get_paginator('list_stacks')
+    stacks = paginator.paginate(StackStatusFilter=stack_status,
+                                PaginationConfig = {'MaxItems': 10000}).build_full_result()['StackSummaries']
```

Comment:
Well, with Paginators I can explicitly set the page size to be more than 100 items. I cannot find an explicit guarantee there that there will be only one page, as build_full_result returns the NextToken in the source code and there is no documentation.